sgl-kernel/build.sh

```diff
@@ -4,24 +4,27 @@ set -ex
 PYTHON_VERSION=$1
 CUDA_VERSION=$2
 PYTHON_ROOT_PATH=/opt/python/cp${PYTHON_VERSION//.}-cp${PYTHON_VERSION//.}
-ENABLE_SM90A=$(( ${CUDA_VERSION%.*} >= 12 ? ON : OFF ))
+DOCKER_IMAGE="pytorch/manylinux2_28-builder:cuda${CUDA_VERSION}"
+ENABLE_SM90A=$( (( ${CUDA_VERSION%.*} >= 12 )) && echo "ON" || echo "OFF" )
 
-if [ ${CUDA_VERSION} = "12.8" ]; then
-   DOCKER_IMAGE="pytorch/manylinux2_28-builder:cuda${CUDA_VERSION}"
+if [ "${CUDA_VERSION}" = "12.8" ]; then
    TORCH_INSTALL="pip install --no-cache-dir --pre torch --index-url https://download.pytorch.org/whl/nightly/cu${CUDA_VERSION//.}"
+   ENABLE_SM100A=ON
 else
-   DOCKER_IMAGE="pytorch/manylinux-builder:cuda${CUDA_VERSION}"
    TORCH_INSTALL="pip install --no-cache-dir torch==2.5.1 --index-url https://download.pytorch.org/whl/cu${CUDA_VERSION//.}"
+   ENABLE_SM100A=OFF
 fi
 
 docker run --rm \
-   -v $(pwd):/sgl-kernel \
-   ${DOCKER_IMAGE} \
+   -v "$(pwd)":/sgl-kernel \
+   "${DOCKER_IMAGE}" \
    bash -c "
+   /sgl-kernel/replace_ptxas.sh ${CUDA_VERSION} && \
    ${PYTHON_ROOT_PATH}/bin/${TORCH_INSTALL} && \
    ${PYTHON_ROOT_PATH}/bin/pip install --no-cache-dir ninja setuptools==75.0.0 wheel==0.41.0 numpy uv && \
    export TORCH_CUDA_ARCH_LIST='7.5 8.0 8.9 9.0+PTX' && \
    export CUDA_VERSION=${CUDA_VERSION} && \
+   export CMAKE_ARGS='-DSGL_KERNEL_ENABLE_SM90A=${ENABLE_SM90A} -DBUILD_FA3=${ENABLE_SM90A} -DSGL_KERNEL_ENABLE_SM100A=$ENABLE_SM100A' && \
```

Comment:
Since FA3 requires 12.3 due to wgmma, I think we no longer need this condition `ENABLE_SM90A`. Can you help delete it from the `-D` options?